frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature

```diff
@@ -0,0 +1,139 @@
+Feature: Upload JAR file
```

Comment:
Please add @add-flow before the feature description

Reply:
Updated